### PR TITLE
docs: Polish SkyWalking doc and a tiny thing about Zipkin

### PR DESCRIPTION
**Pull request by wu-sheng**
### What this PR does / why we need it:
Neither Zipkin nor SkyWalking is a part of OpenTracing ecosystem, especially when we speak from Nginx and LUA perspectives. Also, OpenTracing is being abandoned, their APIs are not widely used, and merged into OpenTelemetry, which also doesn't relate to the Nginx ecosystem. So, let's focus on the tech itself.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,8 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
 
 - **OPS friendly**
 
-  - OpenTracing: support [Apache Skywalking](docs/en/latest/plugins/skywalking.md) and [Zipkin](docs/en/latest/plugins/zipkin.md)
+  - Zipkin tracing: [Zipkin](docs/en/latest/plugins/zipkin.md)
+  - Open source APM: support [Apache Skywalking](docs/en/latest/plugins/skywalking.md)
```
Comment (juzhiyuan) [inline README.md:120]:
```suggestion
  - Open source APM: support [Apache SkyWalking](docs/en/latest/plugins/skywalking.md)
```

File: docs/zh/latest/plugins/skywalking.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ title: skywalking
 
 ## 名字
 
-`Skywalking`(https://github.com/apache/skywalking) 是一个开源的服务跟踪插件。
+`Skywalking`(https://github.com/apache/skywalking) 使用其原生的Nignx LUA tracer，提供分布式追踪，拓扑分析，以及服务和URI级别的metrics信息。
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/skywalking.md:36]:
```suggestion
`Skywalking`(https://github.com/apache/skywalking) 使用其原生的 Nginx LUA tracer，提供分布式追踪、拓扑分析以及服务和 URI 级别的 metrics 信息。
```

**Comment by wu-sheng** [comment]
Why CI get canceled?

**Comment by juzhiyuan** [comment]
> Why CI get canceled?

See https://github.com/apache/apisix/tree/master/.github/actions

**Comment by wu-sheng** [comment]
> > Why CI get canceled?
> 
> See https://github.com/apache/apisix/tree/master/.github/actions

SkyWalking has this, but never cancels CI for the latest commit. What do I miss?

**Comment by juzhiyuan** [comment]
> > > Why CI get canceled?
> > 
> > 
> > See https://github.com/apache/apisix/tree/master/.github/actions
> 
> SkyWalking has this, but never cancels CI for the latest commit. What do I miss?

got it. 😳 You mean the last commit's checks are cancelled unexpectedly, need @Yiyiyimu to confirm.

**Comment by juzhiyuan** [comment]
AFAIK, we have `paths_ignore` in most checks for `docs` changes, do they have a relationship? 🤔

File: docs/en/latest/plugins/skywalking.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ title: skywalking
 
 ## Name
 
-[**Skywalking**](https://github.com/apache/skywalking) is an OpenTracing plugin.\
+[**Skywalking**](https://github.com/apache/skywalking) uses its native Ningx LUA tracer to provide tracing, topology analysis, and metrics from service and URI perspective.\
```
Comment (Yiyiyimu) [inline docs/en/latest/plugins/skywalking.md:36]:
```suggestion
[**Skywalking**](https://github.com/apache/skywalking) uses its native Nginx LUA tracer to provide tracing, topology analysis, and metrics from service and URI perspective.\
```

**Comment by Yiyiyimu** [comment]
> AFAIK, we have `paths_ignore` in most checks for `docs` changes, do they have a relationship? 🤔

Since this PR also changes README, so all CI would be running. Still checking the reason.

**Review by tokers — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**